content/pt-br/docs/tasks/administer-cluster/kubelet-credential-provider.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Configurar um provedor de credenciais de imagem kubelet.
+reviewers:
+- liggitt
+- cheftako
+description: Configure o plugin de provedor de credenciais de imagem do kubelet.
+content_type: task
+min-kubernetes-server-version: v1.26
+weight: 120
+---
+
+{{< feature-state for_k8s_version="v1.26" state="stable" >}}
+
+<!-- overview -->
+
+A partir do Kubernetes v1.20, o kubelet pode recuperar dinamicamente as credenciais para um registro de imagem de contêiner usando plugins executáveis. O kubelet e o plugin de execução se comunicam por meio de stdio (stdin, stdout e stderr) usando APIs versionadas do Kubernetes. Esses plugins permitem que o kubelet solicite credenciais para um registro de contêiner dinamicamente, em vez de armazenar credenciais estáticas no disco. Por exemplo, o plugin pode se comunicar com um servidor de metadados local para recuperar credenciais de curta duração para uma imagem que está sendo puxada pelo kubelet.
+
+Você pode estar interessado em usar essa capacidade se alguma das condições abaixo for verdadeira:
```

Comment:
Is there a better word you could use here instead of "capacidade"?
Perhaps "funcionalidade"? I feel like "capability" doesn't translate very well to Portuguese.

Reply:
agree that "funcionalidade" sounds better